docs/source/getting-started.rst

```diff
@@ -1,48 +1,246 @@
 Getting Started
 ===============
 
-Perform the following steps on your laptop or development machine
+It only takes a few minutes to get StreamAlert up and running!
 
-Dependencies
-------------
+Install Dependencies
```

Comment:
It is implied that these commands are tested in mac os. It would be nice if we add a comment here.

Reply:
Good idea